src/functionalTest/resources/features/F-011 GET Linked Hearing Group/F-011.feature

```diff
@@ -0,0 +1,37 @@
+@F-011
+Feature: F-011: Search for linked hearing group
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+    Given a user with [an active profile in CCD]
+
```

Comment:
missing an AC for processing point 

Check that the provided {id} is an extant/active linkedGroupDetails.requestId - return 404 if not

Reply:
Sorry the final AC was actually returning a 404 - updated the name - for the GET there is 2 ACs one for returning a 404 on non extant requestID and one for incorrect details (but with the GET the only incorrect detail that can return a 400 is a non extant requestID based on spec) - so merged them into a single tests